### PR TITLE
Add validation to user and vehicle model

### PR DESCRIPTION
**Pull request by mickborr**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -4,4 +4,5 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   has_many :vehicles, dependent: :destroy
+  validates :email, :password, :nick_name, presence: true
```
Comment (i-need-coffee) [inline app/models/user.rb:7]: I think it'd be good if we also add a validation for :nick_name, uniqueness: true

File: app/models/vehicle.rb [hunk 2]
```diff
@@ -1,3 +1,8 @@
 class Vehicle < ApplicationRecord
   belongs_to :user
+  validates :category, inclusion: { in:
```
Comment (mjobin123) [inline app/models/vehicle.rb:3]: Hello, I think we should have it in English, will be difficult to change later.

**Review by mjobin123 — COMMENTED** [review]
see comment on language, what are others views??

File: app/models/vehicle.rb [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+    ["trottinette", "trottinette électrique", "vélo", "vélo électrique",
```
Comment (i-need-coffee) [inline app/models/vehicle.rb]: ok nice categories, in english you can write : motorbike, electric motorbike, bicycle, electric bicycle, scooter, electric scooter, kick scooter, electric kick scooter

**Review by i-need-coffee — CHANGES_REQUESTED** [review]

File: app/models/user.rb (hunk 1 above)
Comment (mickborr) [reply app/models/user.rb:7]: yes good idee i did it

File: app/models/vehicle.rb (hunk 3 above)
Comment (mickborr) [reply app/models/vehicle.rb]: thanks perfect i will change it in english

**Pull request merged**